release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,18 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 29 May 2024 - Enhancement - Popup buttons [ID_39010]
+
+Buttons in the popups have been updated to the new style.
+
+#### 29 May 2024 - Fix - Admin - email validation [ID_39791]
+
+The email validation has been updated to support all domain extensions.
```

Comment:
As this is phrased as an enhancement, I'd mention this as an enhancement rather than a fix:

```suggestion
#### 29 May 2024 - Enhancement - Admin - email validation [ID_39791]

Email validation in the Admin app has been updated to support all domain extensions.
```

If you do want to mention it as a fix, you'll instead need to adjust the description so it describes what went wrong from the perspective of the user.

Reply:
It still needs to be described as a fix. I changes the sentence.